src/views/catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/components/AddBootableVolumeButton/components/AddBootableVolumeModal/AddBootableVolumeModal.tsx

```diff
@@ -16,12 +16,14 @@ import CapacityInput from '@kubevirt-utils/components/CapacityInput/CapacityInpu
 import { removeByteSuffix } from '@kubevirt-utils/components/CapacityInput/utils';
```

Comment:
i think that some functions can be extracted to utiil? what do u think?

Reply:
I agree with that, I think this PR is already quite big so maybe can you name which functions other than this one you had in mind and I'll create a separate PR for moving them to utils, wdyt?